SITCOMTN-076.tex

```diff
@@ -191,14 +191,15 @@ \subsection{Summary}
 
 \end{enumerate}
 
-Derived data products that represent visit, coadd, and difference images from ComCam and LSSTCam on-sky commissioning, with the exception of alert postage stamp images, are embargoed for a period of at least 30 days following the observation.
+Derived data products that represent visit, coadd, and difference images from ComCam and LSSTCam on-sky commissioning are embargoed for a period of at least 30 days following the observation, with the exception of alert postage stamp images, and certain specifically approved types of images that illustrate technical performance aspects of the observatory (Section \ref{special_classes}).
```

Comment:
add non-sky as above

Reply:
Not adding "are not on-sky images" because the "specifically approved types of images" listed in the draft are images of the sky, but are taken in non-standard modes (donut images for AOS commissioning, pinhole images) or cover an extremely limited field of view (few arcsecond scale) to illustrate some aspect of technical performance.

Removed a comma for clarity.